modules/calib3d/test/test_solvepnp_ransac.cpp

```diff
@@ -455,7 +770,7 @@ TEST(Calib3d_SolvePnPRansac, input_type)
     EXPECT_LE(cvtest::norm(t1, t4, NORM_INF), 1e-6);
 }
 
-TEST(Calib3d_SolvePnP, double_support)
+TEST(Calib3d_SolvePnPRansac, double_support)
```

Comment:
Not sure of the validity of this test (same thing with `TEST(Calib3d_SolvePnPRansac, input_type)`).
Not sure about how the images points are defined. Since we just check that we got the same results between float and double input, I guess it is more or less OK?